x-kiosk-imx8/Dockerfile

```diff
@@ -0,0 +1,230 @@
+COPY imx-gpu-viv_6.4.3.p1.4-aarch64-r0_arm64.deb /
+
+RUN dpkg -i libgal-imx_6.4.3.p1.4-aarch64-r0_arm64.deb
+RUN dpkg -i libg2d1_6.4.3.p1.4-r0_arm64.deb
+RUN dpkg -i libdrm2_2.4.102.imx-r0_arm64.deb 
+RUN dpkg -i imx-gpu-viv_6.4.3.p1.4-aarch64-r0_arm64.deb
+
+RUN mkdir -p /usr/lib/chromium/
+
+COPY --from=build /chromium/chromium-96.0.4664.110/out/Default/chrome /usr/lib/chromium/
+COPY --from=build /chromium/chromium-96.0.4664.110/out/Default/*.bin /usr/lib/chromium/
+COPY --from=build /chromium/chromium-96.0.4664.110/out/Default/icudtl.dat /usr/lib/chromium/
+COPY --from=build /chromium/chromium-96.0.4664.110/out/Default/chrome-wrapper /usr/lib/chromium/
+RUN ln -s /usr/lib/chromium/chrome-wrapper /usr/bin/chrome
+
+RUN mkdir -p /share/icons/hicolor/16x16/apps/
+RUN mkdir -p /share/icons/hicolor/22x22/apps/
+RUN mkdir -p /share/icons/hicolor/24x24/apps/
+RUN mkdir -p /share/icons/hicolor/32x32/apps/
+RUN mkdir -p /share/icons/hicolor/48x48/apps/
+RUN mkdir -p /share/icons/hicolor/64x64/apps/
+RUN mkdir -p /share/icons/hicolor/128x128/apps/
+RUN mkdir -p /share/icons/hicolor/256x256/apps/
+
+COPY --from=build /chromium/chromium-96.0.4664.110/chrome/app/theme/default_100_percent/chromium/product_logo_16.png /share/icons/hicolor/16x16/apps/chromium.png
+COPY --from=build /chromium/chromium-96.0.4664.110/chrome/app/theme/default_100_percent/chromium/product_logo_name_22.png /share/icons/hicolor/22x22/apps/chromium.png
+COPY --from=build /chromium/chromium-96.0.4664.110/chrome/app/theme/chromium/product_logo_24.png /share/icons/hicolor/24x24/apps/chromium.png
+COPY --from=build /chromium/chromium-96.0.4664.110/chrome/app/theme/default_100_percent/chromium/product_logo_32.png /share/icons/hicolor/32x32/apps/chromium.png
+COPY --from=build /chromium/chromium-96.0.4664.110/chrome/app/theme/chromium/product_logo_48.png /share/icons/hicolor/48x48/apps/chromium.png
+COPY --from=build /chromium/chromium-96.0.4664.110/chrome/app/theme/chromium/product_logo_64.png /share/icons/hicolor/64x64/apps/chromium.png
+COPY --from=build /chromium/chromium-96.0.4664.110/chrome/app/theme/chromium/product_logo_128.png /share/icons/hicolor/128x128/apps/chromium.png
+COPY --from=build /chromium/chromium-96.0.4664.110/chrome/app/theme/chromium/product_logo_256.png /share/icons/hicolor/256x256/apps/chromium.png
+
+COPY --from=build /chromium/chromium-96.0.4664.110/out/Default/*.pak /usr/lib/chromium/
+COPY --from=build /chromium/chromium-96.0.4664.110/out/Default/resources.pak  /usr/lib/chromium/resources.pak 
+
+RUN mkdir -p /usr/lib/chromium/locales/
+COPY --from=build /chromium/chromium-96.0.4664.110/out/Default/locales/*.pak  /usr/lib/chromium/locales/
+
+COPY --from=build /chromium/chromium-96.0.4664.110/out/Default/chrome_crashpad_handler  /usr/lib/chromium/
+
+ENV WAYLAND_USER="weston"
+ENV XDG_RUNTIME_DIR="/run/user/63"
+ENV WAYLAND_DISPLAY="wayland-0"
+
+# For local x86_64 testing
+ENV FIO_HOME /home/weston
+ENV FIO_USERID 63
+ENV FIO_USERNAME weston
```

Comment:
I think you could move these ENV statements as well as the USER and WORKDIR below near the top of this. That will prevent these layers from having to be re-downloaded every time chromium changes

Reply:
done!